index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>| Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="mobile-main">
+        <div class="mobile-header">
+            <div class="app-bar">
+                <div class="logo">
+                    My Logo
+                </div>
+                <div class="menu">
+                    <img src="/Union1.png" alt="">
+                </div>
+            </div>
+            <div class="header-main-content">
```

Comment:
Looks good 😄 but to make it look even better the container of the section should be a `section` tag instead of a `div`.
  * It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info. 🔗